website/routes/CommonFeatures.lazy.tsx

```diff
@@ -342,61 +345,54 @@ function CommonFeatures() {
     });
   }, [rows, sortColumns]);
 
-  const gridElement = (
-    <DataGrid
-      rowKeyGetter={rowKeyGetter}
-      columns={columns}
-      rows={sortedRows}
-      defaultColumnOptions={{
-        sortable: true,
-        resizable: true
-      }}
-      selectedRows={selectedRows}
-      onSelectedRowsChange={setSelectedRows}
-      onRowsChange={setRows}
-      sortColumns={sortColumns}
-      onSortColumnsChange={setSortColumns}
-      topSummaryRows={summaryRows}
-      bottomSummaryRows={summaryRows}
-      className="fill-grid"
-      direction={direction}
-    />
-  );
+  function handleExportToCsv() {
+    flushSync(() => {
+      setIsExporting(true);
+    });
+
+    exportToCsv(gridRef.current!.element!, 'CommonFeatures.csv');
+    setIsExporting(false);
```

Comment:
Should we `flushSync` here too? If the export is synchronous then we can potentially avoid a split second of displaying the grid with virtualization disabled. Not a huge concern though.

If the export is async then we should `await` it.